cmd/containerd-shim-runhcs-v1/main.go

```diff
@@ -64,6 +64,7 @@ func panicRecover() {
 func etwCallback(sourceID guid.GUID, state etw.ProviderState, level etw.Level, matchAnyKeyword uint64, matchAllKeyword uint64, filterData uintptr) {
 	if state == etw.ProviderStateCaptureState {
 		dumpStacks(false)
+		callGCSDumpStacks()
```

Comment:
We should just instead here call `svc.DiagStacks(context.Background(), ...)`. And enlighten `DiagStacks` to handle a `nil` receiver and just do nothing.

Then have it call through to the `host` if one exits and join the `StacksResponse` with `Stacks` and `GuestStacks` then we are set!